frontend/lib/ui/Feedback/Feedback.stories.tsx

```diff
@@ -33,10 +35,8 @@ export const MultipleWarnings: Story = () => {
 
 export const SingleServerError: Story = () => {
```

Comment:
Can you move this test to a new file next to ErrorModal.tsx since it doesn't test the Feedback component anymore.

Reply:
Like this? https://github.com/kiesraad/abacus/pull/471/commits/92e23d3400883762662829565dede231c23dee8f